admin/client/config.ts

```diff
@@ -155,9 +155,12 @@ export async function updateConfig(
   return (await res.json()).data as ConfigFormType;
 }
 
-export async function fetchModels(provider: string): Promise<string[]> {
+export async function fetchModels(
+  provider: string,
+  providerUrl: string,
+): Promise<string[]> {
   const res = await fetch(
-    `${getBaseURL()}/api/management/config/models?provider=${provider}`,
+    `${getBaseURL()}/api/management/config/models?provider=${provider}&provider_url=${providerUrl}`,
```

Comment:
don't you need `encodeURIComponent` to encode the GET parameter?
```suggestion
    `${getBaseURL()}/api/management/config/models?provider=${provider}&provider_url=${encodeURIComponent(providerUrl)}`,
```

Reply:
just a base url so i think it'll alright. If you change to encodeURIComponent then you also need to decode it in python